src/main/java/dev/jbang/source/Project.java

```diff
@@ -152,7 +152,7 @@ public Project setJavaVersion(String javaVersion) {
 		return this;
 	}
 
-	@Nullable
+	@Nonnull
```

Comment:
are we representing null as empty string now or ?

Reply:
The code didn't change, it's just to mark that the return value can never be `null`, so the previous annotation was wrong.